tracing-core/src/span.rs

```diff
@@ -195,6 +197,11 @@ impl<'a> Attributes<'a> {
     pub fn is_empty(&self) -> bool {
         self.values.is_empty()
     }
+
+    /// Returns the field set in this `Attributes`.
```

Comment:
Maybe worth expanding on this to explain that the `FieldSet` includes _all_ of the field names defined by the span's metadata. Some of them may not be recorded by the `Attributes`' `ValueSet` --- they may be empty and recorded later.

How about:
```suggestion
    /// Returns the set of all [fields] defined by this span's [`Metadata`].
    ///
    /// Note that the [`FieldSet`] returned by this method includes *all* the
    /// fields declared by this span, not just those with values that are recorded
    /// as part of this set of `Attributes`. Other fields with values not present in 
    /// this `Attributes`' value set may record values later. 
    ///
    /// [fields]: crate::field
    /// [`Metadata`]: crate::metadata::Metadata
    /// [`FieldSet`]: crate::field::FieldSet
```